pkg/microservice/aslan/core/workflow/service/workflow/pipeline_validation.go

```diff
@@ -733,22 +786,27 @@ func validateServiceContainer2(namespace, envName, productName, serviceName, con
 	}
 
 	for _, p := range pods {
-		// need to filter by serviceName because same container name may exist in multi services
-		if source == setting.SourceFromHelm {
-			annotation := p.Annotations
-			if len(annotation) == 0 {
-				continue
-			}
-			if chartRelease, ok := annotation[setting.HelmReleaseNameAnnotation]; !ok {
-				continue
-			} else if util.ExtraServiceName(chartRelease, namespace) != serviceName {
-				continue
-			}
-		}
-
 		pod := wrapper.Pod(p).Resource()
 		for _, c := range pod.ContainerStatuses {
 			if c.Name == container || strings.Contains(c.Image, container) {
```

Comment:
remove `c.Name == container` and `strings.Contains(c.Image, container)` is not enough for image check